CHANGELOG.md

```diff
@@ -257,6 +257,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
       messages after a successful write to Postgres, DynamoDB, and ES.
     - Updated `@cumulus/api/lib/writeRecords/write-pdr` to publish SNS
       messages after a successful write to Postgres, DynamoDB, and ES.
+  - **CUMULUS-2733**
+    - Updated `_writeGranuleFiles` function to set granule error to be an errors object that
+      contains a stringified list of errors.
```

Comment:
Very open to suggestions on the wording of this

Reply:
I think the implementation detail about it being a stringified list is less important than nothing that `_writeGranuleFiles` now creates an aggregate error containing the workflow error (if any) and the error from writing the granule files (if any).